### DISCUSSION TITLE
How to edit/set/custom the `default` text in swagger_ui?

### DISCUSSION THREAD

**Discussion by pymongo** [poem-web/poem#44] [Q&A] [answered]
How to change the `default` in swagger_ui display?

the `#[OpenApi]` proc macro doen't have arg to set the title instead of "default"?

https://github.com/poem-web/poem/blob/0b697a31cc41194088dc61caf6443579d399d6cc/poem-openapi-derive/src/api.rs#L23-L27

![Screenshot_20211001_162325](https://user-images.githubusercontent.com/42347370/135589047-9a358ce6-ace1-40f4-bedd-ec468d7b7008.png)

**Comment by sunli829** [comment, marked as answer]
This is a tag.

1. Create Tags.

https://github.com/poem-web/poem/blob/0b697a31cc41194088dc61caf6443579d399d6cc/examples/openapi/users-crud/src/main.rs#L10

2. Set the tag of the API.

https://github.com/poem-web/poem/blob/0b697a31cc41194088dc61caf6443579d399d6cc/examples/openapi/users-crud/src/main.rs#L85
